tests/test_utils.py

```diff
@@ -169,3 +169,13 @@ def test_format_cond(condition: str, true_value: str, false_value: str, expected
         expected: The expected output string.
     """
     assert utils.format_cond(condition, true_value, false_value) == expected
+
```

Comment:
I can't seem to put this comment where it belongs on line 9. I think we should add a test for snake case with a _ coming first for example  (_hover)

Reply:
Good idea, added a test for this